### PR TITLE
docs(site): Updating FAQs

### PR DESCRIPTION
**Pull request by basldfalksjdf**

### CODE REVIEW

**Comment by basldfalksjdf** [comment]
@jkowalski I looked at the source code and I think this is correct, but please confirm this tidbit:

>> Before changing your password, you must be connected to your repository. This means that you can reset your password if you forget your password AND you are still connected to your repository. But this also means that you cannot reset your password if you forget your password and you are NOT still connected to your repository, because you will need your current password to connect to the repository.

**Comment by julio-lopez** [comment]
> @jkowalski I looked at the source code and I think this is correct, but please confirm this tidbit:
> 
> > > Before changing your password, you must be connected to your repository. This means that you can reset your password if you forget your password AND you are still connected to your repository. But this also means that you cannot reset your password if you forget your password and you are NOT still connected to your repository, because you will need your current password to connect to the repository.

This seems correct. The wording may be slightly confusing, but we can merge and refine if needed.

**Review by julio-lopez — APPROVED** [review]

File: site/content/docs/FAQs/_index.md [hunk 1]
```diff
@@ -34,6 +46,43 @@ Files/directories are restored from the snapshots you create. To restore the dat
...
+
+[Data deduplication](../features/#backup-files-and-directories-using-snapshots) is enabled automatically by Kopia for all repositories, regardless of whether you use the GUI or CLI. You do not need to do anything.
+
+#### How Do I Change My Repository Password?
+
+You must use Kopia CLI if you want to change your `repository` password; changing password is not currently supported via Kopia GUI. The [`kopia repository change-password` command](../reference/command-line/common/repository-change-password/) is used to change your password. 
+
+Before changing your password, you must be [connected to your `repository`](../getting-started/#connecting-to-repository). This means that you **can** reset your password if you forget your password AND you are still connected to your `repository`. But this also means that you **cannot** reset your password if you forget your password and you are NOT still connected to your `repository`, because you will need your current password to connect to the `repository`.
+
+As a reminder, your `repository` password is used to [encrypt](../features/#end-to-end-zero-knowledge-encryption) your snapshots, so make sure to select a secure password!
```
Comment (julio-lopez) [inline site/content/docs/FAQs/_index.md:84]: Technically, the password is used to derive a key to encrypt the repository primary encryption key (from which encryption keys are derived). However, that ends up being confusing.

What about:

```suggestion
Remember to select a secure _repository password_. The password is used to [decrypt](../features/#end-to-end-zero-knowledge-encryption) and access the data in your snapshots.
```

**Comment by julio-lopez** [comment]
Thanks!

**Pull request merged**